src/EncompassRest/Contacts/BusinessContactLicense.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+
+namespace EncompassRest.Contacts
+{
+    public sealed class BusinessContactLicense : IDirty
```

Comment:
What were your thoughts on renaming this to just `ContactLicense`?

Reply:
it is specific to business contacts. not generic

Reply:
I suppose you're right, it's just confusing to have a `BusinessContact.PersonalContactLicense` property of type `BusinessContactLicense` when there's also a `BusinessContact.BusinessContactLicense` property.

Reply:
i realize it is confusing, but it matches the layout correctly.  I suppose with it not being implemented in the ```BorrowerContact```s, it shouldn't make a difference if we do rename it

Reply:
I'm okay with it either way.

Reply:
then as it is more "technically" correct, i vote leave it.